app/models/participant_profile/ecf.rb

```diff
@@ -105,6 +105,10 @@ def deferred_for?(cpd_lead_provider:)
       !!latest_induction_record_for(cpd_lead_provider:)&.training_status_deferred?
     end
 
+    def matches_lead_provider?(cpd_lead_provider:)
+      relevant_induction_record(lead_provider: cpd_lead_provider&.lead_provider).present?
```

Comment:
@leandroalemao don't use this method, it's sorting by `InductionRecord#start_date` when it should be `InductionRecord#created_at`

Reply:
We should really have one way to do this. If `#relevant_induction_record` isn't doing the right thing can we fix/delete it?

(not one for this PR @leandroalemao, more of a general comment - perhaps worth adding to the backlog)